policy/DecamMapper.yaml

```diff
@@ -167,20 +167,20 @@ calibrations:
     template: cpFLAT/%(calibDate)s/%(filter)s/FLAT-%(calibDate)s.fits[%(ccdnum)d]
     validRange: true
   defects:
-    columns:
-    - date
-    - ccdnum
-    filter: false
+    columns: date
```

Comment:
It's not clear to me why `ccdnum` is no longer needed in `columns`. Is this because each defect is already sorted into ccd by its file path location in `obs_decam_data`?

Reply:
To be honest, this is a gen2 magic I don't really understand.  I assume it looks up ccdnum based on the other entries in the dataId.

Reply:
Put another way, can you leave `ccdnum` in the `columns` and have everything still work? That would be my preference, in case changing our gen2 magic incantations has unintended side effects.

Reply:
I'm happy to leave it if tests pass.